packages/seacas/scripts/exodus3.in.py

```diff
@@ -851,12 +851,41 @@ def copy(self, fileName, include_transient=False, mode='a'):
                                           ctypes.byref(self.io_ws),
                                           EX_API_VERSION_NODOT)
 
-        self.__copy_file(fileId, include_transient)
+        self.copy_file(fileId, include_transient)
         EXODUS_LIB.ex_close(fileId)
 
         return exodus(fileName, mode)
 
 
+    #
+    # copy to a new already created database
+    #
+    # --------------------------------------------------------------------
+    def copy_file(self, file_id, include_transient=False):
+        """
+        Copies exodus database to the database pointed to by `fileId`
+        Does not return anything.
+
+        >>> exofile = exodus.exodus(file_name, mode='w')
+        >>> exo_copy = exo.copy_file(exofile.fileId, include_transient=True)
```

Comment:
I think it would be more consistent to change this to a with  statement as I believe the rest of the documentation has been changed to prefer that.

Reply:
Thanks.  I will update.